### PR TITLE
fix(rust): remove if_let from indent captures

### PR DESCRIPTION
**Pull request by kyazdani42**
fixes https://github.com/nvim-treesitter/nvim-treesitter/issues/3073

### CODE REVIEW

**Review by clason — APPROVED** [review]

**Comment by kyazdani42** [comment]
let me add a test before merging :)

**Comment by clason** [comment]
Oh, I'm not merging anything -- I learned to leave highlighting decisions up to @theHamsta :D

**Comment by kyazdani42** [comment]
@theHamsta is this good to go ?

**Pull request merged**
